packages/framework/src/Component/CustomerUploadedFile/CustomerUploadedFileFacade.php

```diff
@@ -264,7 +267,7 @@ public function getByIdSlugExtensionAndCustomerUserOrHash(
         ?CustomerUser $customerUser = null,
         ?string $hash = null,
     ): CustomerUploadedFile {
-        if (!$hash && !$customerUser) {
+        if (!$hash && !$customerUser && !$this->administratorFrontSecurityFacade->isAdministratorLogged()) {
```

Comment:
just a thought - maybe we can extract this into a method that will be used here and in `CustomerUploadedFileController` so we avoid duplicating the condition? Something like `CustomerUploadedFileFacade::isAccessToFileAllowed($hash, $customerUser): bool`

Reply:
https://github.com/shopsys/shopsys/pull/3388/commits/112552928f76d33d066294e50348fd325e451158